### ISSUE TITLE
Boards break after the server is started

### ISSUE THREAD

**Issue by KakEgorka** [mattermost/mattermost#23023]
After assembling the server and launching the boards. The boards disappear after a few minutes. An error appears in the console
Safari: 
<img width="1320" alt="Снимок экрана 2023-04-19 в 11 42 20" src="https://user-images.githubusercontent.com/109951790/232993728-a0ba62e2-564b-4fc5-8730-00406628e288.png">

Chrome:
<img width="1006" alt="Снимок экрана 2023-04-19 в 11 46 09" src="https://user-images.githubusercontent.com/109951790/232993773-272ea1a3-f231-4980-9f7d-f3ea5d7e97fa.png">

Mattermost Teams: v7.10.0

**Comment by calebroseland** [comment]
@KakEgorka is this happening just recently? There were some recent breaking changes that require a small change.  https://community.mattermost.com/core/pl/hzfrkmysh7gxixs65c4pebpahc

> Module version is upgraded to 8.0.
tl;dr: All server related code is under server directory.
Please delete your existing go.work file under the repo root. The make run-server command will auto-generate them in the new place.



I ran into boards and playbooks not loading yesterday while switching between branches, and deleting go.work ended up working for me.

**Comment by KakEgorka** [comment]
I tried today. Re-cloned and tried again. Boards and playbooks are also taught.

**Comment by calebroseland** [comment]
@KakEgorka what command are you using to run MM locally? `make run`? Are there any more details you can share? You said it's disappearing after a few minutes—does it work at first for a short time?

**Comment by KakEgorka** [comment]
> @KakEgorka what command are you using to run MM locally? `make run`? Are there any more details you can share? You said it's disappearing after a few minutes—does it work at first for a short time?

Yes, I use run to run.
And yes, the boards work first. Then when you click somewhere, MM hangs on a white screen and then opens again, but there are no boards anymore.

Last time I tried to go into user settings. The screen went white, the page reloaded and the boards disappeared.

**Comment by KakEgorka** [comment]
Also in the console errors on react-dom 
`[Error] Warning: Unknown event handler property `onWidthChange`. It will be ignored.
div
div
PureComponent@
AutosizeTextarea
div
PureComponent@
QuickInput
div
PureComponent@
Connect(SuggestionBox)
Connect(SuggestionBox)
div
PureComponent@
Connect(Textbox)
Connect(Textbox)
div
div
div
AdvanceTextEditor
form
PureComponent@
Connect(AdvancedCreatePost)
div
div
PureComponent@
Connect(ChannelView)
withRouter(Connect(ChannelView))
PureComponent@
Connect(ChannelIdentifierRouter)
withRouter(Connect(ChannelIdentifierRouter))
Component@
Component@
div
div
PureComponent@
Connect(CenterChannel)
withRouter(Connect(CenterChannel))
div
div
ChannelController
Component@
TeamController
Connect(Component)
Suspense
TeamController
PureComponent@
Connect(LoggedIn)
Suspense
LoggedIn
Component@
LoggedInRoute
Component@
Fe
ThemeProvider
CompassThemeProvider
Component@
PureComponent@
PureComponent@
Connect(IntlProvider)
RootProvider
PureComponent@
Connect(Root)
Suspense
Root
Component@
Component@
Provider
PureComponent@

	printWarning (react-dom.development.js:67)
	error (react-dom.development.js:43)
	validateProperty$1 (react-dom.development.js:3448)
	warnUnknownProperties (react-dom.development.js:3559)
	validateProperties$2 (react-dom.development.js:3583)
	validatePropertiesInDevelopment (react-dom.development.js:8765)
	setInitialProperties (react-dom.development.js:9041)
	finalizeInitialChildren (react-dom.development.js:10201)
	completeWork (react-dom.development.js:19470)
	completeUnitOfWork (react-dom.development.js:22807)
	performUnitOfWork (react-dom.development.js:22782)
	workLoopSync (react-dom.development.js:22702)
	renderRootSync (react-dom.development.js:22665)
	performSyncWorkOnRoot (react-dom.development.js:22288)
	performSyncWorkOnRoot
	unstable_runWithPriority (scheduler.development.js:468)
	flushSyncCallbackQueueImpl (react-dom.development.js:11322)
	flushSyncCallbackQueue (react-dom.development.js:11309)
	scheduleUpdateOnFiber (react-dom.development.js:21888)
	dispatchAction (react-dom.development.js:16139)
	dispatchAction`

and and error with /api/v4/trial-license/prev
<img width="1328" alt="Снимок экрана 2023-04-21 в 17 10 00" src="https://user-images.githubusercontent.com/109951790/233632522-921731c8-30b5-4832-b1a5-56ac3e5fd1db.png">

**Comment by hmhealey** [comment]
Hi @KakEgorka. Are you running MM locally on `http://localhost:8065`, or do you have it running on a different URL? Or are you perhaps using Gitpod?

In development, we run separate Webpack dev servers for Boards and Playbooks, and it seems like they can't be reached because either:
1. They're just not running. That shouldn't be the case though if you're using `make run` or `make run-client` to run the web app. It should error out if those failed to start.
2. Those servers are unreachable from the browser. If you have a proxy or Firewall in front of MM, you'll need to open ports 9006 and 9007 and set the `MM_BOARDS_DEV_SERVER_URL` and `MM_PLAYBOOKS_DEV_SERVER_URL` environment variables to override the location of those servers. For example, I have my development server running on `https://harrison-mm`, so I've had to set these environment variables for my server to work locally:
    ```bash
    export MM_SERVICESETTINGS_SITEURL="https://harrison-mm"
    export MM_BOARDS_DEV_SERVER_URL="https://harrison-mm:9006"
    export MM_PLAYBOOKS_DEV_SERVER_URL="https://harrison-mm:9007"
    ```

**Comment by KakEgorka** [comment]
> Hi @KakEgorka. Are you running MM locally on `http://localhost:8065`, or do you have it running on a different URL? Or are you perhaps using Gitpod?
> 
> In development, we run separate Webpack dev servers for Boards and Playbooks, and it seems like they can't be reached because either:
> 
> 1. They're just not running. That shouldn't be the case though if you're using `make run` or `make run-client` to run the web app. It should error out if those failed to start.
> 2. Those servers are unreachable from the browser. If you have a proxy or Firewall in front of MM, you'll need to open ports 9006 and 9007 and set the `MM_BOARDS_DEV_SERVER_URL` and `MM_PLAYBOOKS_DEV_SERVER_URL` environment variables to override the location of those servers. For example, I have my development server running on `https://harrison-mm`, so I've had to set these environment variables for my server to work locally:
>    ```shell
>    export MM_SERVICESETTINGS_SITEURL="https://harrison-mm"
>    export MM_BOARDS_DEV_SERVER_URL="https://harrison-mm:9006"
>    export MM_PLAYBOOKS_DEV_SERVER_URL="https://harrison-mm:9007"
>    ```

Hi @hmhealey! I use running MM locally on http://localhost:8065 . I launch it via `make run` or `make run-client`. Also I don't use proxy or Firewall.

I will try to run it with your recommendations and let you know how it will work. Thanks

**Comment by hmhealey** [comment]
Hmm, that's strange. Based on that, everything should work straight out of the box. What do you get if you go to http://localhost:9006/remote_entry.js in your browser then? The error messages you posted look like the the dev server for Boards and Playbooks aren't reachable, but perhaps the error is something else. This'll help confirm that the server is running and should be visible to the main MM web app.

You should see something like this when visiting the above URL
![image](https://user-images.githubusercontent.com/3277310/236024930-3b60bbba-bb5e-422d-8147-139e9b9f4e8b.png)
